### PR TITLE
Feature/lol/adjust land cover keys nlcd

### PR DESCRIPTION
**Pull request by lliss**
Connects #765 

Builds off of https://github.com/WikiWatershed/model-my-watershed/pull/803

Updates the keys used to pass data to the geoprocessor and tr55.

Note: this requires that you have TR55 running off of this branch: https://github.com/lliss/tr-55/tree/feature/lol/align-nlcd-strings2 or you won't get model output.

 * Update the keys used to pass data to the geoprocessor.
 * Update tests.
 * Recreate the microsite csv data.
 * Remove legacy key translations.

### CODE REVIEW

**Comment by lliss** [comment]
Once TR55 update is in, there will be one more commit here to bump the TR55 version in the requirements file.

File: src/mmw/apps/water_balance/templates/home_page/index.html [hunk 1]
```diff
@@ -208,88 +208,88 @@ <h1 class="infil">0.0 cm</h1>
...
+                    <li id="thumb-developed_open">
+                        <a href="#land-developed_open" aria-controls="filter-tab" role="tab" data-toggle="tab">
+                            <img src="{% static 'images/water_balance/thumb_turfGrass.png' %}" alt="Developed, Open Space" data-container="body" data-toggle="popover" data-placement="left" data-title="Developed, Open Space" data-content="Areas with a mixture of some constructed materials, but mostly vegetation in the form of lawn grasses. Impervious surfaces account for less than 20% of total cover. These areas most commonly include large-lot single-family housing units, parks, golf courses, and vegetation planted in developed settings for recreation, erosion control, or aesthetic purposes.">
                         </a>
-                        <label>HIR</label>
+                        <label>Developed, Open Space</label>
```
Comment (rajadain) [inline src/mmw/apps/water_balance/templates/home_page/index.html]: Consider renaming to "Developed-Open" for consistency.

**Comment by rajadain** [comment]
How can I run TR-55 from the specified branch?

**Comment by jamesmcclain** [comment]
> How can I run TR-55 from the specified branch?

You can either change this line https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/requirements/base.txt#L12 and reprovision, or you can ssh into the `worker` and `app` VMs and `pip uninstall tr55` then `pip install` from github using these instructions:  https://stackoverflow.com/questions/8247605/configuring-so-that-pip-install-can-work-from-github .

File: src/mmw/apps/water_balance/templates/home_page/index.html (hunk 1 above)
Comment (lliss) [inline src/mmw/apps/water_balance/templates/home_page/index.html]: Good catch.

Comment (lliss) [inline src/mmw/apps/water_balance/templates/home_page/index.html]: Actually, I just revisited the original issue, and while they do give acceptable shortened versions for "Developed, Low (Medium and High) Intensity" they don't give once for open space. So in the interest of sticking to the request as written, I'll leave this. It's easy to revise later and we may end up shortening more of these labels so we can do them all at once.

**Comment by lliss** [comment]
Now incorporates https://github.com/WikiWatershed/model-my-watershed/pull/799

**Comment by rajadain** [comment]
@jamesmcclain thanks. These are the commands I ran to install the custom TR-55 module:

```
$ vagrant ssh app -c 'sudo pip uninstall tr55 && sudo pip install git+git://github.com/lliss/tr-55.git@feature/lol/align-nlcd-strings2'
$ vagrant ssh worker -c 'sudo pip uninstall tr55 && sudo pip install git+git://github.com/lliss/tr-55.git@feature/lol/align-nlcd-strings2'
```

On an unrelated note, it is unfortunate how the repo is named `tr-55` but the package is named `tr55`.

**Comment by rajadain** [comment]
+1 everything works very well:

image

All tests are passing.

**Comment by mmcfarland** [comment]
Taking a look now...

File: src/mmw/apps/home/templates/home/model.html [hunk 2]
```diff
@@ -132,7 +132,7 @@ <h3 class="light">Project Name</h3>
                                 </li>
                                 <li>
                                     <div class="thumb"></div>
-                                    <label class="dark">Row Crop</label>
+                                    <label class="dark">Cultivated Crop</label>
```
Comment (rajadain) [inline src/mmw/apps/home/templates/home/model.html]: Consider changing to "Cultivated Crops" (plural)

**Comment by mmcfarland** [comment]
+1 Looks good!

**Comment by lliss** [comment]
@azavea-bot rebuild

**Pull request merged**